src/custom-elements/src/components/date-picker/date-picker.tsx

```diff
@@ -14,6 +14,13 @@ import customParseFormat from 'dayjs/plugin/customParseFormat';
 import { ChiStates, CHI_STATES } from '../../constants/states';
 import { ESCAPE_KEYCODE } from '../../constants/constants';
 
+type Time = {
```

Comment:
This could be moved to `constants/types.ts` in case it can be used later on a different component

Reply:
And I'd ask also to move ESCAPE_KEYCODE import with the rest of the constants import (nothing to do with your changes) 😄

Reply:
Done.